src/main/java/bookbob/functions/CommandHandler.java

```diff
@@ -171,30 +171,33 @@ public void list(Records records) {
         }
     }
 
-    // @@Author G13nd0n
+    // @@author G13nd0n
     public void delete(String nric, Records records) throws IOException {
+        assert nric != null:
+                "Please provide a valid NRIC";
+
```

Comment:
Good use of assertion to assert nric != null when user wants to delete a specific patient record